src/main/kotlin/dev/supergrecko/kllvm/core/typedefs/Builder.kt

```diff
@@ -8,31 +8,40 @@ import org.bytedeco.llvm.LLVM.LLVMBuilderRef
 import org.bytedeco.llvm.LLVM.LLVMValueRef
 import org.bytedeco.llvm.global.LLVM
 
-public class Builder internal constructor(internal val llvmBuilder: LLVMBuilderRef) : AutoCloseable, Validatable, Disposable {
+public class Builder public constructor(context: Context = Context.getGlobalContext()) : AutoCloseable, Validatable, Disposable {
```

Comment:
Because the Builder has a public API which offers a parameter with default value Kotlin will generate a public parameterless constructor. In this scenario there is no reason need to have the internal parameterless one.